src/structures/TextChannel.js

```diff
@@ -27,7 +27,7 @@ class TextChannel extends GuildChannel {
     this.topic = data.topic;
 
     /**
-     * If the Discord considers this channel NSFW
+     * If the guild considers this channel NSFW
```

Comment:
Wouldn't this make more sense as `If this channel is marked (as?) NSFW`? (idk if the `as` is needed)

Reply:
no